python_backend/api/v1/endpoints/quality.py

```
@@ -125,18 +154,26 @@ async def analyze_company_text_quality(
+        # 성능 로깅
+        execution_time = time.time() - start_time
+        logger.info(f"기업용 분석 완료 - 실행시간: {execution_time:.2f}초")
+
+        # 성능 경고 (5초 이상 소요시)
+        if execution_time > 5.0:
+            logger.warning(f"느린 품질분석 호출 감지: {execution_time:.2f}초 (텍스트 길이: {len(request.text)})")
+
         return response
-        
+
     except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)